src/classes/TestAmountToText.cls

```diff
@@ -0,0 +1,72 @@
+/**
+ * Created by ociszek001 on 21.05.2019.
+ */
+@isTest
+private class TestAmountToText {
+
+    @TestSetup
+    static void loadInvoices() {
+        User u = TestDataClass.insertUser();
+        List<Invoice__c> inv = new List<Invoice__c>();
+        System.runAs(u) {
+            Test.startTest();
+            VAT__c vat = TestDataClass.insertVat();
+            inv = TestDataClass.loadInvoices();
+            List<Service__c> ser = TestDataClass.insertServices(new List<VAT__c>{
+                    vat
+            }, inv);
+            Test.stopTest();
+        }
+        System.assertEquals(false, inv.isEmpty());
+    }
+    @isTest
+    static void TestAmountToText() {
+        User u = [SELECT Id from User WHERE userName = :TestDataClass.userName];
```

Comment:
Nie zwróciłem wcześniej na to uwagi bo zapomniałem Ci powiedzieć ale w przypadku jak nie znajdzie takiego user'a to się wywali bardzo brzydko. Żeby temu zapobiec zamień wszystkie tego typu select'y na taki:

https://salesforce.stackexchange.com/questions/75515/best-approach-way-to-query-single-sobject-record